### PR TITLE
Update the embedding.md file

### PR DESCRIPTION
**Pull request by amanp592**
Solves the issue as mentioned in the tensorflow repository [#27612](https://github.com/tensorflow/tensorflow/issues/27612)

### CODE REVIEW

**Comment by amanp592** [comment]
@lamberta  @MarkDaoust kindly review this PR

File: site/en/guide/embedding.md [hunk 1]
```diff
@@ -211,11 +211,14 @@ vectors in the embedding variable, except for the header.  Consequently, the
...
-format.
+format.Also, when you open the load data tab in the embedding visualizer there
+it is specified about what is TSV file of vector and what is the TSV file
+of metadata with an example.
 
 To use images as metadata, you must produce a single
 [sprite image](https://www.google.com/webhp#q=what+is+a+sprite+image),
-consisting of small thumbnails, one for each vector in the embedding.  The
+consisting of small thumbnails,  one for each vector in the embedding.
+This single image contain all the images the dataset has.The
```
Comment (lamberta) [inline site/en/guide/embedding.md]: > This single image contain all the images the dataset has.

Can you clarify this?

File: site/en/guide/embedding.md [hunk 2]
```diff
@@ -211,11 +211,14 @@ vectors in the embedding variable, except for the header.  Consequently, the
 variable.  If the TSV metadata file has only a single column, then we don’t
 expect a header row, and assume each row is the label of the embedding. We
 include this exception because it matches the commonly-used "vocab file"
-format.
+format.Also, when you open the load data tab in the embedding visualizer there
+it is specified about what is TSV file of vector and what is the TSV file
+of metadata with an example.
```
Comment (lamberta) [inline site/en/guide/embedding.md]: Clean up. Is this accurate:
The embedding visualizer's *Load Data* tab specifies the TSV file for the vector and metadata.

File: site/en/guide/embedding.md (hunk 1 above)
Comment (amanp592) [reply site/en/guide/embedding.md]: @lamberta  yeah i did something wrong there, i was suppose to write that a sprite image is a collection of all sample images into one single image.

File: site/en/guide/embedding.md (hunk 2 above)
Comment (amanp592) [reply site/en/guide/embedding.md]: i'll correct this

**Comment by lamberta** [comment]
This change was moved to https://github.com/tensorflow/docs/pull/474

**Pull request closed without merging**